transforms/code/code2parquet/transform.config

```diff
@@ -0,0 +1,20 @@
+#
+# This is intended to be included across the Makefiles provided within
+# a given transform's directory tree,  so must use compatible syntax.
+#
+################################################################################
+# This defines the name of the transform and is used to match against
+# expected files and is used to define the transform's image name. 
+TRANSFORM_NAME=code2parquet
+
+################################################################################
+# This defines the transforms' version number as would be used
+# when publishing the wheel.  In general, only the micro version
+# number should be advanced relative to the DPK_VERSION. 
+#
+# If you change the versions numbers, be sure to run "make set-versions" to 
+# update version numbers across the transform (e.g., pyproject.toml).
+CODE2PARQUET_PYTHON_VERSION=$(DPK_VERSION)
```

Comment:
@daw3rd How are we getting DPK_VERSION if we are not including .make.versions ?

Reply:
The transformer Makefiles include `$(REPOROOT)/transforms/.make.transforms` it includes `$(REPOROOT)/.make.defaults` and the last one includes `$(REPOROOT)/.make.versions`